src/pages/costModels/components/inputs/selector.tsx

```diff
@@ -94,6 +94,7 @@ const SelectorBase: React.FunctionComponent<SelectorProps> = ({
         variant={SelectVariant.single}
         placeholderText={placeholderText}
         aria-label={ariaLabel}
+        direction={direction}
```

Comment:
Could you use PatternFly's `SelectDirection` type here?

`direction={isDirectionUp ? SelectDirection.up : SelectDirection.down}`